extensions/sovity-messenger/src/main/java/de/sovity/edc/extension/messenger/controller/SovityMessageController.java

```diff
@@ -108,10 +113,10 @@ public Response post(
         }
     }
 
-    private SovityMessageResponse processMessage(SovityMessageRequest compacted, Handler<Object, Object> handler) throws JsonProcessingException {
+    private SovityMessageResponse processMessage(SovityMessageRequest compacted, HandlerWithClaims<Object, Object> handler, ClaimToken claims) throws JsonProcessingException {
```

Comment:
isn't there in java a `HandlerWithClaims<*, *>` or `HandlerWithClaims<?, ?>` for when you explicitly don't want to care about the contents